packages/console/src/consts/tenants.ts

```diff
@@ -43,4 +43,4 @@ export const getCallbackUrl = (tenantId?: string) =>
 
 export const getSignOutRedirectPathname = () => (isCloud ? '/' : ossConsolePath);
 
-export const maxFreeTenantNumbers = 3;
+export const maxFreeTenantNumbers = 10;
```

Comment:
also i'm thinking this should be listed in a specific api instead of hard-coding in both frontend and backend. or at least to keep a single source of truth..

Reply:
Per offline discussion, we can put the const in @logto/schemas so that console/cloud/integration-tests can all relies on this, which satisfies SSOT.

Reply:
Good